pandas/core/resample.py

```diff
@@ -240,6 +240,7 @@ def aggregate(self, arg, *args, **kwargs):
         result, how = self._aggregate(arg, *args, **kwargs)
         if result is None:
             result = self._groupby_and_aggregate(arg,
+                                                 None,
```

Comment:
Could you explain why you need this `None` argument?

Reply:
@gfyoung 

Using test scenario, `multiplier` arg will be passed to `_groupby_and_aggregate` as `grouper` arg if we omit `None`. So it's better to pass `None` explicitly.

Reply:
can you then pass grouper as a kwarg

Reply:
we *may* need to rename this arg itself to avoid a user pass ing ``grouper=`` (unlikely though)

Reply:
> can you then pass grouper as a kwarg

I can't. 

```
TypeError: _groupby_and_aggregate() got multiple values for argument 'grouper'
```

The workaround for this will be something like that:
```
grouper = None
result = self._groupby_and_aggregate(arg,
                                     grouper,
                                     *args,
                                     **kwargs)
```

Reply:
> we may need to rename this arg itself to avoid a user pass ing grouper= (unlikely though)

There can be the same behaviour for `how` arg:

```
TypeError: _groupby_and_aggregate() got multiple values for argument 'how'
```

@jreback 
Is there any util that check `kwargs` for allowed parameters?